programs/clearing_house/src/math/bank_balance.rs

```diff
@@ -210,3 +213,44 @@ pub fn get_balance_value(
     let (value, _) = get_balance_value_and_token_amount(bank_balance, bank, oracle_price_data)?;
     Ok(value)
 }
+
+pub fn check_withdraw_limits(bank: &Bank) -> ClearingHouseResult<bool> {
+    let deposit_token_amount =
+        get_token_amount(bank.deposit_balance, bank, &BankBalanceType::Deposit)?;
+    let borrow_token_amount =
+        get_token_amount(bank.borrow_balance, bank, &BankBalanceType::Borrow)?;
+
+    let max_borrow_token = bank.withdraw_guard_threshold.max(
+        (deposit_token_amount / 6)
```

Comment:
can lend out up to 90% of deposits (giving 10% buffer reserves for depositors who wish to withdrawal) 

its a chained max, min statement (allow up to 1/6th, twap friction otherwise, and stop at 9/10th)

Reply:
ok this is the initial amount we can lend out, do we want to to put it on the bank or do you think 16% makes sense for all banks?